doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/knowledge-store-rest-api-requests-proc.adoc

```diff
@@ -141,19 +142,20 @@ For curl utility:
 * `-u`: Enter the user name and password of the {CENTRAL} user with the `rest-all` role.
 * `-H`: Set the following headers:
 ** `Accept`: `application/json`
+** `Accept-Language`: `en-US`
```

Comment:
Can we add the default case info here? wdyt?

Reply:
Done, have added a note.